deploy/quickstart.sh

```diff
@@ -17,7 +18,7 @@ openssl req -new -x509 -batch -nodes -days 11000 \
         -subj "/O=io.interconnectedcloud/CN=qdr-white.sa-telemetry.svc.cluster.local" \
         -out qdr-server-certs/tls.crt \
         -keyout qdr-server-certs/tls.key
-oc create secret tls qdr-white-cert --cert=qdr-server-certs/tls.crt --key=qdr-server-certs/tls.key
+oc create secret tls qdr-white-cert --cert=qdr-server-certs/tls.crt --key=qdr-server-certs/tls.key || true
```

Comment:
If you're trying to make this script easy to re-run, then instead of a straight `oc create` here (and on L28), you could try a technique like this: `kubectl create ... --dry-run=true -o yaml | kubectl apply -f -` This lets you construct a new resource with the create command, but still apply it as a patch. It will create the resource if it does not exist, or apply the new config to any existing resource, while leaving any additional config that may have been added intact.

I don't think you can do this with a project, though.